.travis.yml

```diff
@@ -6,18 +6,12 @@ python:
 # Ideally we would be pip installing tk-core, but we need the testing framework from core,
 # so we will clone the repo install to a known location and run the tests there.
 install:
-    - git clone https://github.com/shotgunsoftware/tk-core /tmp/tk-core
-    - pushd /tmp/tk-core
-    - git checkout master
-    - popd
+    - git clone --branch SG-13264-python-3-compat --depth 1 https://github.com/shotgunsoftware/tk-core /tmp/tk-core
     # install pyside - from https://stackoverflow.com/questions/24489588
     - sudo apt-get install libqt4-dev
     - pip install PySide --no-index --find-links https://parkin.github.io/python-wheelhouse/;
     # Travis CI servers use virtualenvs, so we need to finish the install by the following
     - python ~/virtualenv/python${TRAVIS_PYTHON_VERSION}/bin/pyside_postinstall.py -install
-    - pip install coverage
-    - pip install coveralls
 
 # command to run tests
-script: /tmp/tk-core/tests/run_tests.sh --test-root tests --with-coverage
-after_success: coveralls
+script: python /tmp/tk-core/tests/run_tests.py --test-root tests
```

Comment:
The python 3 on tk-core tries to run tests with both Python 2 and 3, but we only want to run with 2.